docker-compose.yml

```diff
@@ -277,6 +277,22 @@ services:
       - _APP_SMTP_USERNAME
       - _APP_SMTP_PASSWORD
 
+  appwrite-maintenance:
+    entrypoint: maintenance
+    container_name: appwrite-maintenance
+    build:
+      context: .
+    restart: unless-stopped
+    networks:
+      - appwrite
+    depends_on:
+      - redis
+      - mariadb
+    environment:
+      - _APP_ENV
+      - _APP_REDIS_HOST
+      - _APP_REDIS_PORT
+
```

Comment:
Something wierd I noticed in the appwrite-maintenance container. 
Even though I have not passed `_APP_MAINTENANCE_INTERVAL` environment variable in the docker-compose, Im able to access it in `maintenance.php`

Reply:
@eldadfux

Reply:
Did you define a default value in the main Dockerfile?

Reply:
https://github.com/appwrite/appwrite/pull/766/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R97

This one

Reply:
So that's the value you get.

Reply:
If any env variable that you declare in the main docker file becomes visible in the services , then what is the significance of lines https://github.com/appwrite/appwrite/pull/766/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R291-R294

Reply:
It allow us to edit the vars using the .env file, we used to edit them from the docker-compose.yml in the past, but the env allow as to reuse vars and avoid duplicated values in the main file.